Loop/Views/SimpleBolusView.swift

```diff
@@ -0,0 +1,353 @@
+//
+//  SimpleBolusView.swift
+//  Loop
+//
+//  Created by Pete Schwamb on 9/23/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+import LoopKit
+import LoopKitUI
+import HealthKit
+
+struct SimpleBolusView: View, HorizontalSizeClassOverride {
+
+    @Environment(\.dismiss) var dismiss
+    
+    @State private var shouldBolusEntryBecomeFirstResponder = false
+    @State private var isKeyboardVisible = false
+    @State private var isClosedLoopOffInformationalModalVisible = false
+
+    var displayMealEntry: Bool
+    @ObservedObject var viewModel: SimpleBolusViewModel
+    
+    init(displayMealEntry: Bool, viewModel: SimpleBolusViewModel) {
+        self.displayMealEntry = displayMealEntry
+        self.viewModel = viewModel
+    }
+    
+    var title: String {
+        if displayMealEntry {
+            return NSLocalizedString("Simple Meal Calculator", comment: "Title of simple bolus view when displaying meal entry")
+        } else {
+            return NSLocalizedString("Simple Bolus Calculator", comment: "Title of simple bolus view when not displaying meal entry")
+        }
+    }
+        
+    var body: some View {
+        GeometryReader { geometry in
+            VStack(spacing: 0) {
+                List() {
+                    self.infoSection
+                    self.summarySection
+                }
+                // As of iOS 13, we can't programmatically scroll to the Bolus entry text field.  This ugly hack scoots the
+                // list up instead, so the summarySection is visible and the keyboard shows when you tap "Enter Bolus".
+                // Unfortunately, after entry, the field scoots back down and remains hidden.  So this is not a great solution.
+                // TODO: Fix this in Xcode 12 when we're building for iOS 14.
```

Comment:
[nit] Do we want to log a Jira ticket for this? Or is there one for general "search and resolve all iOS 14 related TODOs"?